react-native-purchases-ui/src/index.tsx

```diff
@@ -155,6 +157,57 @@ type InternalFooterPaywallViewProps = FooterPaywallViewProps & {
   onMeasure?: ({height}: { height: number }) => void;
 };
 
+export type CustomerCenterManagementOption = 
+  | 'cancel'
+  | 'custom_url'
+  | 'missing_purchase'
+  | 'refund_request'
+  | 'change_plans'
+  | 'unknown'
+  | string; // This is to prevent breaking changes when the native SDK adds new options
+
+  export interface CustomerCenterCallbacks {
+  /**
+   * Called when a feedback survey is completed with the selected option ID.
+   */
+  onFeedbackSurveyCompleted?: ({feedbackSurveyOptionId}: { feedbackSurveyOptionId: string }) => void;
+  
+  /**
+   * Called when the manage subscriptions section is being shown.
+   */
+  onShowingManageSubscriptions?: () => void;
+  
+  /**
+   * Called when a restore operation is completed successfully.
+   */
+  onRestoreCompleted?: ({customerInfo}: { customerInfo: CustomerInfo }) => void;
+  
+  /**
+   * Called when a restore operation fails.
+   */
+  onRestoreFailed?: ({error}: { error: PurchasesError }) => void;
+  
+  /**
+   * Called when a restore operation starts.
+   */
+  onRestoreStarted?: () => void;
+
+  /**
+   * Called when a refund request starts with the product identifier.
```

Comment:
Maybe worth mentioning this is iOS only?
```suggestion
   * Called when a refund request starts with the product identifier. iOS-only callback.
```